### PR TITLE
Task/integ journey account

### PR DESCRIPTION
**Pull request by EvenModiguy**

### CODE REVIEW

File: journey/src/main/java/com/kisio/navitia/sdk/ui/journey/presentation/model/DisruptionMessageRoadmapModel.java [hunk 1]
```diff
@@ -10,8 +10,8 @@
     private String disruptionAvoidanceMessage;
     private String disruptionAvoidanceActionMessage;
 
-    public DisruptionMessageRoadmapModel(Context context, int sectionType) {
-        super(context, sectionType);
+    public DisruptionMessageRoadmapModel(int sectionType) {
```
Comment (nicolas-chaix) [inline journey/src/main/java/com/kisio/navitia/sdk/ui/journey/presentation/model/DisruptionMessageRoadmapModel.java]: is 'sectionType' and IntDef? should we add an annotation?

File: journey/src/main/java/com/kisio/navitia/sdk/ui/journey/presentation/model/SectionRoadmapModel.java [hunk 2]
```diff
@@ -1,28 +1,13 @@
...
-import android.graphics.Typeface;
-
-import com.kisio.navitia.sdk.ui.journey.util.Font;
-
 public abstract class SectionRoadmapModel {
     private int sectionType;
-    private Typeface typeface;
 
-    public SectionRoadmapModel(Context context, int sectionType) {
+    public SectionRoadmapModel(int sectionType) {
```
Comment (nicolas-chaix) [inline journey/src/main/java/com/kisio/navitia/sdk/ui/journey/presentation/model/SectionRoadmapModel.java]: same here: IntDef / Annotation?

File: journey/src/main/java/com/kisio/navitia/sdk/ui/journey/util/Format.java [hunk 3]
```diff
@@ -357,28 +354,27 @@ public static String duration(Context context, Integer seconds, boolean useFullF
...
-            case WALKING:
-                return stepDuration < 60 ? context.getResources().getString(R.string.less_than_a_minute_walk) :
-                        String.format(context.getResources().getString(R.string.a_time_walk),
-                                duration(context, stepDuration, false));
-            case PARK:
-                return duration(context, stepDuration, false);
-            default:
-                return "";
+        if (SectionMode.RIDESHARING.getModeName().equalsIgnoreCase(stepMode)) {
+            return stepDuration < 60 ? context.getResources().getString(R.string.less_than_a_minute_ridesharing) :
```
Comment (nicolas-chaix) [inline journey/src/main/java/com/kisio/navitia/sdk/ui/journey/util/Format.java]: should we use a constant for '60'?

File: journey/src/main/res/drawable/background_time_arrival_solid.xml [hunk 4]
```diff
@@ -6,5 +6,4 @@
     <corners
         android:bottomRightRadius="@dimen/cards_corner_radius"
         android:topRightRadius="@dimen/cards_corner_radius" />
-
 </shape>
```
Comment (nicolas-chaix) [inline journey/src/main/res/drawable/background_time_arrival_solid.xml]: missing end line

File: journey/src/main/java/com/kisio/navitia/sdk/ui/journey/presentation/model/DisruptionMessageRoadmapModel.java (hunk 1 above)
Comment (EvenModiguy) [reply journey/src/main/java/com/kisio/navitia/sdk/ui/journey/presentation/model/DisruptionMessageRoadmapModel.java]: Nope, there's no `intDef`, only values for Constants. We should make this refactoring when kotlin will pass by.

**Pull request merged**